### PR TITLE
CAL-124 Refactored Test Class names to follow DDF conventions

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR refactors all test classes in Alliance to follow a postfix pattern (*Test.java)
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@troymohl @glenhein 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl
@jlcsmith
#### How should this be tested?
Build DDF
#### Checklist:
- [ ] Documentation Updated
- [X] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/CatalogMgrImplTest.java [hunk 1]
```diff
@@ -22,9 +22,7 @@
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
-import java.util.ArrayList;
```
Comment (jlcsmith) [inline catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/CatalogMgrImplTest.java]: I'm curious why these imports changed with this PR.

**Comment by coyotesqrl** [comment]
👍

File: catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/CatalogMgrImplTest.java (hunk 1 above)
Comment (bdeining) [inline catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/CatalogMgrImplTest.java]: These were unused.  The NSILI generated IDL does not adhere to checkstyle, so it has to be turned off.

**Comment by jlcsmith** [comment]
👍

**Pull request merged**